packages/mongodb-memory-server-core/src/util/db_util.ts

```diff
@@ -25,8 +25,8 @@ export function getHost(uri: string): string {
 /**
  * Basic MongoDB Connection string
  */
-export function getUriBase(host: string, port: number, dbName: string): string {
-  return `mongodb://${host}:${port}/${dbName}?`;
+export function uriTemplate(host: string, port: number, dbName: string, query?: string): string {
+  return `mongodb://${host}:${port}/${dbName}` + (!isNullOrUndefined(query) ? `?${query}` : '');
```

Comment:
should i change `query` to be an array of strings, to do something like `uriTemplate(ip, port, dbName, ['option1=1', 'option=2'])` or should it stay `uriTemplate(ip, port, dbName, 'option1=1&option=2')`?

Reply:
There is no purpose to do it right now. It can be done after publishing v7.0.0.

But if you want to do something crazy with a connection string, then you may see the following gist with parser/serializer https://gist.github.com/nodkz/d2bb4e91ba8ca9fd3178d45b33d94357

According to [_parseOptions mthod](https://gist.github.com/nodkz/d2bb4e91ba8ca9fd3178d45b33d94357#file-connectionstring-ts-L149) options are represented by Object, not array.

For example, we are using this parser for creating ENV variables for some systems which are not support connection strings, e.g for [redis-sentinel](https://gist.github.com/nodkz/753ef827d0083c377a944e0f7e468bcf)

Reply:
this is an bit overkill for this project (at least at the moment), so i thought either `option1=1&option2=2` for now, and ask if it should be change to an array `['option1=1', 'option2=2']` (or maybe an object)
-> so if i understand correctly, i can let it stay like it is atm, and change when needed otherwise

Reply:
According to our current [config options](https://github.com/nodkz/mongodb-memory-server#available-options-for-mongomemoryserver) we accept `instance.args` as array. In such a way, `object` isn't good shape for us if we don't want to change its type.

So if we decide to provide static typings for all available instance's args then `object` will be good. BUT we work with different major mongod versions, which have different args. And making args statically typed can be an unsolved job. Now I come to a resolution that `object` for `options` is not suitable for us.

So `string` or `array of strings` is the best form for options. And how I see now an array of strings is more convenient.

Reply:
ok, will change

> And making args statically typed can be an unsolved job

i think we only "officially support" the most common arguments (that are found on many versions, example: `auth`)

Reply:
8dffa64ee4d56523cefd993b1158d2a9e224cd8a changes this to an string-array